ext/nnstreamer/tensor_filter/tensor_filter_tensorrt.cc

```diff
@@ -225,21 +225,34 @@ tensorrt_subplugin::invoke (const GstTensorMemory *input, GstTensorMemory *outpu
   /* Copy input data to Cuda memory space */
   memcpy (_inputBuffer, input->data, input->size);
 
+  /* Initialize output buffer pointer to nullptr for safety */
+  output->data = nullptr;
```

Comment:
why? in allocBuffer, it calls cudaMallocManaged (&output->data, output->size), which effectively does output->data = malloc(). This appears meaningless.